CPC-CHARTER.md

```diff
@@ -295,7 +295,7 @@ from the OpenJS Foundation board.
 **Project**: a technical collaboration effort that is organized through the
 project mentorship process and approved by the TAC.
 
-**Contributors**: contribute code or other artifacts, but do not have the right
+**Contributors**: contribute code or other artefacts, but do not have the right
```

Comment:
"artifacts" is valid American English, but do we have a preference for British spelling?

Reply:
A least in the Node.js project the preference is for US English (I think that is even written into the docs).

Reply:
Ooops - I thought I had successfully ignored all the US/British alternatives and only fixed true typos. This one slipped past. Will revert this one.